javalib/src/main/scala/java/util/Timer.scala

```diff
@@ -0,0 +1,115 @@
+package java.util
+
+import scala.collection._
+import scala.concurrent.duration._
+import scala.scalajs.js.timers._
+
+class Timer(val name: String, val isDaemon: Boolean) {
+  private[util] var cancelled = false
+  private[util] val tasks = mutable.Set[TimerTask]()
+
+  def this() = this("timer", false)
+
+  def this(isDaemon: Boolean) = this("timer", isDaemon)
+
+  def this(name: String) = this(name, false)
+
+  private[util] def scheduleOnce(task: TimerTask, delay: Long): Unit = {
+    tasks += task
+    setTimeout(delay.millis) {
+      task.doRun()
+    }
```

Comment:
Also I think the `task` should be removed from `tasks` upon completion, otherwise there's a memory leak in `tasks`.